src/MoBi.Presentation/Presenter/Main/NotificationPresenter.cs

```diff
@@ -250,11 +249,25 @@ public void ShowContextMenu(IViewItem notificationItem, Point popupLocation)
          contextMenu.Show(_view, popupLocation);
       }
 
+      public void Handle(ShowProjectConversionNotificationsEvent eventToHandle)
+      {
+         Handle(eventToHandle as ShowNotificationsEvent);
+         notifyForUntraceableChanges(eventToHandle.NotificationMessages.Select(x => x.Object as MoBiSimulation).Where(x => x != null && x.HasUntraceableChanges).Distinct().ToList());
```

Comment:
what about we create a local variables here?

Reply:
cannot we have an objec that has the simulation and this flag just for the conversion? instead of havign it into the simulation

Reply:
The simulation is keeping this flag because the sync status remains red in the explorer until it is reconfigured. That's also why you get the reminder each time you commit and not only once during conversion. I'm not sure if that's necessary but it seems right. The simulation is not in sync and it will never be without reconfigure.